CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 # Change Log
 
-## v0.0.2-beta
+## 1.0.0
+
+**Release date:** 2022-01-31
+
+* Improved documentation
+* Changed the defualt values in configuration template
+* Resume the Bamboo server after installation
+* Improved install and uninstall scripts
+* Made the product version, Bamboo software and Bamboo agent resources configurable
+* Improved the end to end tests
+* Changed the default product version (Bamboo software and Bamboo agent Helm charts) to `1.0.0`
```

Comment:
Can we list the product version that we have updated to as well?

Reply:
I removed this line as this change is done as part of version `0.0.2` and already we have it in change log